docs/source/hw_acceleration_tutorial.ipynb

```diff
@@ -0,0 +1,1306 @@
lYLS9YBgWlrDk8uMbR8QmeY9NqtTDtnL3dNluDPrY26XZajPb2WGcZgRPiuwGGZWJYLpXIQJdIsaKuBFqb7Oz0MbXGtS3G4yOEktTC5PjkFVqZ7G7v4tgOg94226NrZPmM2eI1hqEv531dPC8gcD2qckytAg72rzNfTrFp0e92SNMlRZXgBxGtdoso6pDlKWHgI1RJFIc4rsvxyTGtVgvPszCcmG77NvPJGs8JQX5Fnq9w9S4oj8ki49rtO/S7Pb589BghUibTJ9xsZyjt0en67BzEHD2fQG0Qegrf1VS1SVkb/PTvCv7zf1D86PcCfuv3bDqDnB/9QU2Z/RuQ3yGoR5yOhzydvuDmW33aPRPTKqgCxULbYEjQNUWyQOsJW6OAg2strt0acHp8C8uOWW1WJJsN4/ML8qJASUlVOQQxpKmmLDRFnmNbGlodOu0+aZYThDFCwnK1otftoXRNE2ZkEoQ+jhNjm5LecE0cTei2fWqRs0oSuv0hv/9PDjk/m3Hy+pzJ5ATHtvEDn34/ZHu3j2VppCqRovoHqo2B+WvLVwbQ2CQaBpfz2wUajWt7uIGHQTPjXcuauioxtG7cHdAoochFQpakXJwdN24Nrk8Ut2i1u0RxC88LiSKb++/scu++RqmbVHXJrYOc2VRgWprNpqKuTLRsvJ3dwMc0HIQwyPMcqWSjOJsmbzSm3/RzxYUrLlxx4YoLV1z4Zlz4Vl/yfc/nxs3b7O3us143y02ObTOZXjCZnFPXgmvXrvP02fMmGEFZlJWiLCHLKjbrhLKsMLBxHB9pSPyiwtCKzLGIohDXtZjP57z3/rsEgU1ZJmglWW+WVLW4fIozmc0XnJ1eUGQFlmWTpjm2Y7G3v8dkckaeN1v1RVnSpnnZl8slURAwubhAKUW31+Vg5wMCZ06elyRrSVkl1PXTZmGo18U8mhK3fLotTZknuH2TQTdkMvmM0+O36La36Pe+w9bIpD0MkabF559d59rukqNXG6KewvIVVWXh+yEGLi3bI81LkizHV83qSRhFvHx5zJOHj5k8m9NqDxlu7dLr9XFQ+KZGGhLTttjd2yNdb9gsNxjKpio0WV2QpwaRN6TT6dPp9cnTnCePH9HrxdgumMqmyASz6Zof/eh9+r1tzs5fcayfcPzqIVsy4cbNm6wma/xgm5u3b/Pwiyf4XshsnnLDdTncv0adrTFNiWM7mJZJWVfcvXePTqeP1GBqGpVpOcVxCtbJBWFtUNYay67wHJvZ7ISqqnGcDn7YoxIlRbnk+GiBlpLBaJdebx/P20ajkKJESYGWNWWa0unbFNUFQXAfpRX99ggDn8DaZevwHqtkzNnFGc5qRlHkuI6HqBSL9Zput4tlmpyfHbNYjHE9CzyXebLB0StGO3vMZ0MM7dCOWiwWF+xEilAmXLt+G8fU/PLTTwi9kJdPxliWx2jHpq5rZnMH3zYYDlwCvyZNc8xcg2WxyRR//RcZDz8z+PE/Cnj7gYEfzrCcv8Z0u7QKm9dHms9/ucSxK/YOY3Z32rz/gebRQ5vlJEDUNobWqFpQpHMGvW203mI6T9EIpKqJ4gDTMsjzgsUyp7/dZrRrc3FSEbWaJcfVtMIKbIJWm17fRCqB79l8/PHHxO0AAxMD2N/b4vq1W6Dm2O4vmC9K6qJGVhZVXVOJmqoU3Ly5zf5BxMnRGWdHM9arhOUS5ouc/uAWlr1FGILvFRhGgVJZY5eoJQYCUChlXSqKRjPvKA3KMqeySpxLX2QAx7XxPY8yz341v6n1ZUe3gYVSkiJLKbKM6cU5GCa26xKFMXGrTRi1CIIY1wnY3w3Z221mPisRNdArBEUOWaoQQqKkZntrRJJsKKsCIQRSiv9f6vC37Vxx4YoLV1y44sIVF74ZF77Vi7f/+7/6V3/kOi5KKcqi5OLinKPj15RlSRAEbG1t0xv02azXFGWzBNNElCeUpQBMDBo7LgwTyzbxkgRWG4LRAC8K2Kw3dDptBsM+nU6bdruF5zVpdnlW4PkhUilevTpmOl1QVwrbbtwcDg4OiKKATbLh4nxCkVdIIel0uihqbCQH+zaen5JtNKZhE4RtbNtFKklVls2TpaxxHZdev0cYOYSxhe9M0XpJp2/R6tVoY4lh5sRhD8cO8f0B6drm5asjJlPN6asp60WBYVoMdgJs22M+M0nXzf/H9VzqqkBWJb7rMbuY4y5L+sqlNRwSdbt8+eVDvvj8U1aLOaUoWK4XRHGb4XAHU9qcvDzHdzp02yP6nS3u3n6HKIiR1Gw2C5SsGPY7dDshpgFZXuD5MWXZPBnHcRe0ZracM0+WbG/1yJOE3d0DHn75iDBs8eLlK7SGXqeHbdq8dfMO+zt9hoOIMAzp9kcslmvcKOTazVuYjg0mZGmCYRoIocnSCs8NsFyfotpQVglJOmU2GzdLXYbC932qMmcxPSdL5mTZkrRIEaKmKnNCN8B3HGBFt7VNLU9Zrc+xzF1acZ9Ou4+oQGsLy3IJ/MYOLS9SpCoIgxa243C4f0iZC/IsY5OsUUrw85/9jL//6c/AdFivV+T5Go2N7QQUWYlluTiOT12VnLx+jaglcdyiKFLKLGWzTBBFlzK/icUhWZpQC7BMm8HIwDBLXN/hxp0OH/5uj6hr8PnHE148Fjh2RLtv4HgJUatCK5uq1qzmgum54LO/S0CGJGmM5/wWjrtFKeZI2cw5XkwqPvmsZDJekKY5RVayWScslxvStKQuNZPzivFpCVqTrCXruY1pOBwebpEXBbWwCAOPO2/f4L3373N4eMhyVWKZHo5jgZK47gTPe4SQkjL3idoOj78qqSuXycUFVVEz2m4Thh6B7xMEPt1eF89zsCyDPLNYrWxW6xZJElHLEMvu4Lghlm1hmI2lnSnNZm5TaSxMoijGwATVKHZlkaG0RDcRkKAuF8m+tm3TaH0ZxqTf+DdeelnXolnKWyyYTiZMLi5YzKfk2QapJAZg2w6uaxIENlFs0+5YxC2DIAKlUjRNyupimb
```

Comment:
newline

Reply:
well technically, this file was generated by Google Colab